### ISSUE TITLE
Allow user to specify the location of the json files for the extension

### ISSUE THREAD

**Issue by thamara** [thamara/vscode-code-annotation#14]
Currently we use the global storage for the json file for the extension. It would be really nice if this could be configurable.
The idea is to include a configuration that will allow that.

For reference, check `config.ts`.

**Comment by MattBidewell** [comment]
Hey! 
I wouldn't mind picking this up :)

**Comment by MattBidewell** [comment]
#17 PR

**Comment by AleksueiR** [comment]
Any chance this is going to be implemented? You can't share annotations without it.

**Comment by thamara** [comment]
@AleksueiR Yes, that's the intention. But to be completely honest, I haven't got time to dedicate to working on the extension as much as I would like. For sure, there's a lot I want to improve on it, and getting feedback from folks like you help me understand what's the most critical improvements.
But at the end, the day only have 24h. :(

I'm planning to have some time to focus on the extension somewhere, maybe next week or the one after. When this issue is fixed, I'll mention you, so you can get updates.
